tests/double_index_bug_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019-2023 ETH Zurich and the DaCe authors. All rights reserved.
+from sympy import core as sympy_core
+
+import dace
+from dace import subsets
+
+
+def test_double_index_bug():
```

Comment:
Maybe I should put this test somewhere else? :thinking:

Reply:
Probably in tests/passes/scalar_to_symbol_promotion_test.py?